jupyter_scheduler/models.py

```diff
@@ -47,6 +47,8 @@ def __str__(self) -> str:
 
 
 class Status(str, Enum):
+    CREATED = "CREATED"
+    QUEUED = "QUEUED"
```

Comment:
When is the `QUEUED` status getting set? `CREATED` is the default status upon job creation, but I don't see when the `QUEUED` status ever gets set in the server extension. Looks like it's only defined here.

Reply:
We don't support it, but it will be useful for other implementors who will have queueing systems. We might start using for scheduled jobs, but haven't thought about it yet.